packages/components/Notification/src/Notification.styling.ts

```diff
@@ -33,14 +33,18 @@ export const stylingSettings: UseStylingOptions<NotificationProps, NotificationS
       }),
       ['backgroundColor', ...borderStyles.keys, ...layoutStyles.keys],
     ),
-    icon: buildProps(() => ({
-      style: {
-        alignSelf: 'center',
-        marginRight: 16,
-      },
-      height: 24,
-      width: 24,
-    })),
+    icon: buildProps(
+      (tokens: NotificationTokens) => ({
+        style: {
+          alignSelf: 'center',
+          marginRight: 16,
```

Comment:
(As I must ask whenever I see "marginRight" instead of "marginStart"...)
Should this flip in RTL? If so, we probably want to use marginStart?